core/modules/user/components/UserShippingDetails.ts

```diff
@@ -132,13 +125,8 @@ export const UserShippingDetails = {
     fillCompanyAddress () {
       this.useCompanyAddress = !this.useCompanyAddress
       if (this.useCompanyAddress) {
-        let index
-        for (let i = 0; i < this.currentUser.addresses.length; i++) {
-          if (toString(this.currentUser.addresses[i].id) === toString(this.currentUser.default_billing)) {
-            index = i
-          }
-        }
-        if (index >= 0) {
+        const index = this.currentUser.addresses.findIndex((address) => toString(address.id) === toString(this.currentUser.default_billing))
```

Comment:
why not just find() in condition statement? seems clener https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/find

Reply:
good point :) changed